web/modules/custom/joinup_notification/src/JoinupMessageDeliveryInterface.php

```diff
@@ -138,6 +138,19 @@ public function setRecipients(array $accounts): self;
    */
   public function setRecipientsAsEmails(array $mails): self;
 
+  /**
+   * Sets the list of accounts that will be added to bcc.
+   *
+   * @param array $bcc_emails
+   *   A list of user emails.
+   *
+   * @return $this
+   *   Return the object for chaining.
```

Comment:
`@return $this` is a special kind of fish that doesn't need a comment. From https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards#types

> You may omit the description if using `@return $this` or `@return static`.

However, it's not wrong, you can keep it.